app/Launcher/Main.hs

```diff
@@ -38,6 +39,8 @@ import           Data.X509.Extra (failIfReasons, genRSA256KeyPair,
 main :: IO ()
 main = do
 
+  -- _ <- runUpdater updaterData
```

Comment:
Remove this.
We need a simple `FilePath` when somebody wants to call the `runUpdater` - then it will try to run the installer on the `FilePath`.